### ISSUE TITLE
Language Import on uSync 8

### ISSUE THREAD

**Issue by cmwalolo** [KevinJump/uSync8#60]
when importing multiple languages, the import could fail. Possibly the fallback language is not yet imported. Invisible in the report, and mess up totally the content import.

**Comment by KevinJump** [comment]
Hi,

Do you have some steps to reproduce this issue? just wondering is it certain settings, configurations, or order of languages?

in theory, the config file for a piece of content contains all the languages for that property and they are set in one go, but their might be something we are missing. 

Kevin

**Comment by cmwalolo** [comment]
I created english first and French second. English as been set to fallback to French. And this seems to fail. I imported on a. Empty install with en-us added to the languages

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by KevinJump** [comment]
Hi with the latest release (8.3.0) we have made some changes to the way languages are tidyied up (because you can now change a language in the Umbraco UI) this might help with this issue.

**Comment by Szpi** [comment]
Hi, We have similar problem with import of file that supports multi language. The problem could be that when we set "Allow varying by culture" usync sometimes exports config like this : 

```
<backgroundImage>
      <Value><![CDATA[]]></Value>
</backgroundImage>
```

but I believe (I'm not an expert) it should be somewhat like this:
```
<backgroundImage>
      <Value Culture="de"><![CDATA[]]></Value>
</backgroundImage>
```

When importing there is an exception on item.SetValue(...) (see picture below), because culture is empty and it is passed as one of the parameters.
![image](https://user-images.githubusercontent.com/23508122/71520102-b7bf0a00-28ba-11ea-96a9-aeb4f335bd3c.png)


When we publish two language usync file is correct and contain two language that we have in our system.

Also we noticed that Exporting usync (from usync UI) files overrides our configuration and it is without culture. 

It would be very helpful for us if Export functionality would write all language that are defined in the system or maybe is there a better way to to this?
